### DISCUSSION TITLE
Error in InventoryHP for chains containing Th-232

### DISCUSSION THREAD

**Discussion by seb01957** [radioactivedecay/radioactivedecay#81] [General]
I was attempting to benchmark radioactivedecay with other codes I use or have developed.  I noticed using InventoryHP and decaying Cm-248, Pu-240 and U-236 produced erroneous inventories for chain member Th-232 and its progeny for the chains listed above. Th-232 is very close to being a stable radionuclide.  Perhaps that is the problem.  Thanks for any insight.  Sebastian.

**Comment by alexmalins** [comment]
Hi Sebastian, Thank you - this is a bug.

I've checked & there are 3 radionuclides in the high-precision datasets with the wrong half-life due to a bug within the dataset creation notebook. These are `Th-232`, `Sm-147` and `Rb-87`.

I'll work on a fix and release a new radioactivedecay version asap.

Thanks again for reporting the problem !

**Comment by alexmalins** [comment]
Ok this should now be fixed in v0.4.14 that is now on PyPI. Conda-forge will automatically update to the new release within a few hours.

Can you please check if v0.4.14 fixes things in your calculations?

**Comment by seb01957** [comment]
I will test v0.4.14 tomorrow and let you know the results.  Thanks Sebastian.

**Reply by seb01957** [reply]
The errors with chains containing Th-232 is fixed in my calculations.  Thanks Alex.
